test/models/inheritanceClass.ts

```diff
@@ -0,0 +1,44 @@
+import { arrayProp, prop } from '../../src/prop';
+import { getModelForClass, modelOptions } from '../../src/typegoose';
+
+@modelOptions({
+  schemaOptions: {
+    discriminatorKey: 'width',
```

Comment:
this is as far as i can tell not a valid schema Option ( https://mongoosejs.com/docs/guide.html#collection )

Reply:
It seems that the documentation is not accurate. I found an example of the option: 
https://mongoosejs.com/docs/api.html#singlenestedpath_SingleNestedPath-discriminator

And the typings of mongoose also allow this option.

Reply:
ok, thanks, could you now say me why you would want a discriminator in a nested path?

Reply:
i think i understood it now, [thanks to this blog post](https://thecodebarbarian.com/mongoose-4.12-single-embedded-discriminators)

Reply:
PS: i understood it now, but why is this needed in the tests? because i cant find anywhere where a discriminator call is used